code/modules/games/cas.dm

```diff
@@ -20,6 +21,10 @@
 
 /obj/item/weapon/deck/cas/New()
 	..()
+	if (card_face == "cas_white")
+		card_text_list = file2list("config/cas_white.txt")
```

Comment:
I would prefer it if you only loaded from the two files once, maybe at the start of the round?

It is not that important though and you don't have to stop merging this for it.